spec/datadog/tracing/contrib/grpc/datadog_interceptor/server_spec.rb

```diff
@@ -35,6 +35,7 @@
     it { expect(span.get_tag('rpc.system')).to eq 'grpc' }
     it { expect(span.get_tag('rpc.service')).to eq 'My::Server' }
     it { expect(span.get_tag('rpc.method')).to eq 'endpoint' }
+    it { expect(span.get_tag('span.kind')).to eq('server') }
```

Comment:
Missing assertion at[ `without an error handler` ](https://github.com/DataDog/dd-trace-rb/pull/2365/files#diff-6203f701e2b6e456c5ad7061d028c7b941e1f67b7f0a7e66779bf389e8dc139aR75) and [`with an error handler` ](https://github.com/DataDog/dd-trace-rb/pull/2365/files#diff-6203f701e2b6e456c5ad7061d028c7b941e1f67b7f0a7e66779bf389e8dc139aL86)